### PR TITLE
sellerPage

### PR DESCRIPTION
**Pull request by gaganparmar2003**
This is my Seller page code which we will be using as a user login id as a seller. we have to join customer and admin id with that. Let me know you guys are okay with this code or want any further changes.

### CODE REVIEW

**Review by htdguide — APPROVED** [review]
Good job gagan! This need some changes in the future but it is enough for today

File: studio3code/src/SellerPage.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { doc } from 'firebase/firestore';
+import { firestore } from './firebase';
+
```
Comment (Sahil4353) [inline studio3code/src/SellerPage.js:5]: Thank you for submitting the Seller page code with the integrated user login functionality for sellers.

**Review by Sahil4353 — APPROVED** [review]
Thank you for submitting the Seller page code with the integrated user login functionality for sellers.

**Comment by gnijen** [comment]
Fantastic work! The functionality is well-executed and fits the requirements perfectly.

**Pull request merged**